### PR TITLE
[JP] Fixed quote typos, Add some Kai quotes

### PR DESCRIPTION
**Pull request by redpizza**

### CODE REVIEW

File: data/jp/quotes.json [hunk 1]
```diff
@@ -660,7 +660,7 @@
 		"Docking(1)": "まだ航行できますが…ご命令ならば。",
 		"Docking(2)": "じっとしてるのは性に合いませんね。",
 		"Join": "水雷戦隊、出撃します！",
-		"Sortie": "水雷戦隊、出撃します！",
+		"Sortie": "期待に応えてみせます。,
```
Comment (sinsinpub) [inline data/jp/quotes.json]: `"` is missing

Comment (redpizza) [reply data/jp/quotes.json]: Oops, Sorry.

Comment (sinsinpub) [reply data/jp/quotes.json]: checking still not passed; shall not edit on github web, it changes all lines...

Comment (redpizza) [reply data/jp/quotes.json]: I'm sorry for that. I have to learn more how github works...

Comment (redpizza) [reply data/jp/quotes.json]: It's okay at all. probably.

**Pull request merged**